forgotPassword.js

```diff
@@ -1,35 +1,20 @@
-document.getElementById("send_code_btn_fp_pg").addEventListener("click", onSendCodeClick);
+document.getElementById("send_button_fp").addEventListener("click", onSendCodeClick);
 
 function onSendCodeClick() {
-
-    const user_email = document.getElementById("em_input_fp_pg").value;
-    const span_user_email = document.querySelector("#confirm_txt span");
+    document.getElementById("input_email").style.display = "none";
+    document.getElementById("second_step").style.display = "inline";
+    const userEmail = document.getElementById("em_input_fp_pg").value;
+    const spanUserEmail = document.querySelector("#confirm_txt span");
     document.getElementById("confirm_txt").style.display = "inline";
     document.getElementById("confirm_txt").classList.add("animate__animated");
     document.getElementById("confirm_txt").classList.add("animate__fadeInLeft");
```

Comment:
Don't Repeat Yourself. Do you know this [principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)? Here you three times repeat `document.getElementById("confirm_txt")`. Let's add const.

Reply:
Done